internal/cli/usercmd/push.go

```diff
@@ -261,6 +257,24 @@ func (c *EpinioClient) stageLogs(appRef models.AppRef, stageID string) {
 	}()
 }
 
+func routeMessage(msg *termui.Message, routes []string) *termui.Message {
+	if routes == nil {
+		if len(routes) > 0 {
```

Comment:
Hm, I'm not sure. How can routes be nil and `>0`?

Reply:
Will recheck the logic there.

Reply:
Fixed. Commit dd20c7680623195b1487aa257ebed8cb0a8824a2

The logic was inverted. Reworked to reduce indentation and make logic clearer.
This should have a full run of CI to see if there are tests affected by this change to the output.

Reply:
Thank you, much better! :+1: